src/ucp/proto/proto_multi.c

```diff
@@ -202,14 +209,17 @@ ucs_status_t ucp_proto_multi_init(const ucp_proto_multi_init_params_t *params,
                    lane_perf->min_length);
         perf.min_length = ucs_max(perf.min_length, min_length);
 
-        weight_sum          += lpriv->weight;
-        mpriv->min_frag      = ucs_max(mpriv->min_frag, lane_perf->min_length);
-        mpriv->max_frag_sum += lpriv->max_frag;
-        lpriv->weight_sum    = weight_sum;
-        lpriv->max_frag_sum  = mpriv->max_frag_sum;
-        lpriv->opt_align     = ucp_proto_multi_get_lane_opt_align(params, lane);
-        mpriv->align_thresh  = ucs_max(mpriv->align_thresh,
-                                       lpriv->opt_align);
+        weight_sum           += lpriv->weight;
+        min_end_offset       += lane_perf->bandwidth *
+                                context->config.ext.min_rndv_chunk_size /
+                                min_bandwidth;
```

Comment:
minor: 
```c
        min_end_offset       += (lane_perf->bandwidth / min_bandwidth) *
                                context->config.ext.min_rndv_chunk_size;
```

Reply:
maybe nitpick, but i'd keep multiplications before divisions to try to minimize rounding errors, let me know else i can change it.